src/components/dashboard/dashboardItem/ItemBase.js

```diff
@@ -123,46 +124,17 @@ const DashboardItem = ({ item }) => {
                     root: classes.root,
                 }}
             >
-                <Typography color="textSecondary" variant="body2" component="p">
+                <Typography
+                    color="textSecondary"
+                    variant="body2"
+                    component="p"
+                    style={{ overflow: "ellipsis", height: "4.2em" }}
```

Comment:
I think this height can be removed.  Otherwise, it looks like the action buttons start to get cramped as the card resizes:
![image](https://user-images.githubusercontent.com/8909156/100029590-da99bd00-2dae-11eb-81fa-a2ab031c4442.png)

Reply:
Removing height will cause the text to `overflow`. It is required for `ellipsis` to work properly. I have adjusted the `height` and `DESC_MAX_LENGTH` to mitigate this problem. From my testing this happens only at a particular screen size and it should be fixed now.